sdk/eventhubs/azure-messaging-eventhubs/migration-guide.md

```diff
@@ -1,205 +1,170 @@
 
-| In v3 | Equivalent in v5 | Sample |
-|---|---|---|
-| `EventHubClient.createFromConnectionString()` | `var builder = new EventHubClientBuilder().connectionString();`<br/>then either `builder.buildProducerAsyncClient();` or<br/>`builder.consumerGroup().buildConsumerAsyncClient();` | [Publishing events][PublishEventsWithCustomMetadata], [Consuming events][ConsumeEvents] |
-| `EventHubClient.createWithAzureActiveDirectory()` | `var builder = new EventHubClientBuilder().tokenCredential();`<br/>then either `builder.buildProducerAsyncClient();` or<br/>`builder.consumerGroup().buildConsumerAsyncClient();` | [Publishing events with Azure AD][PublishEventsWithAzureIdentity] |
-| `EventProcessorHost.EventProcessorHostBuilder`<br/>`.newBuilder()` | `new EventProcessorClientBuilder().buildEventProcessorClient()` | [EventProcessorClient with Azure Storage Blobs][EventProcessorClientInstantiation] |
+### Client hierarchy and constructors
 
-### Sending events
+In the interest of simplifying the API surface, we've made three clients. One client is for producing events, [EventHubProducerAsyncClient][EventHubProducerAsyncClient] has async operations and [EventHubProducerClient][EventHubProducerClient] is the sync version. The other two clients are intended for reading events. [EventProcessorClient][EventProcessorClient] is what you would typically use to read events from all partitions with an optional, but recommended checkpointing feature. [EventHubConsumerAsyncClient][EventHubConsumerAsyncClient] is what you would use for lower-level control of [EventData][EventData] consumption from a single partition.
 
-`EventHubProducerAsyncClient` and `EventHubProducerClient` can publish events to a single partition or allow the service
-to load balance events between all the partitions. The behaviour is determined when using
-[`CreateBatchOptions`][CreateBatchOptions] in `producer.createBatch(CreateBatchOptions)`.
+Creation of producer and consumer clients is done through the [EventHubClientBuilder][EventHubClientBuilder] while the [EventProcessorClient][EventProcessorClient] is created through the [EventProcessorClientBuilder][EventProcessorClientBuilder].
 
-| In v3 | Equivalent in v5 | Sample |
-|---|---|---|
-| `PartitionSender.send(...)` | `EventHubProducerAsyncClient.send()` | [Publishing events to a specific partition][PublishEventsToSpecificPartition] |
-| `EventHubClient.send(...)` | `EventHubProducerAsyncClient.send()` | [Publishing events][PublishEventsWithCustomMetadata] |
+#### Create an Event Hub producer
 
-### Receiving events
+Event Hub synchronous and asynchronous producers in v3 were created using static methods in `EventHubClient`. In v5, they are created with [EventHubClientBuilder][EventHubClientBuilder]. Samples below demonstrate how to create an Event Hub producer, but the same steps can be used for the consumer.
```

Comment:
> Event Hub synchronous and asynchronous producers in v3 were created using static methods in `EventHubClient

I don't see any "producers" in v3. The send method is either directly on the client for automatic routing or we can create "partition senders" to send to particular partition. I also don't see sync/async variations in any classes in v3. So, how is the above statement accurate?

If we want to show how the constructing a EventHubProducerClient is different from constructing a EventHubClient, we can rename this section to `Instantiating clients` and talk about all 3 clients. See https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/servicebus/azure-messaging-servicebus/migration-guide.md#instantiating-clients for example